### ISSUE TITLE
New editor stylesheet

### ISSUE THREAD

**Issue by paaljoachim** [Getbeans/Beans#333]
Hey

Regarding this issue: https://github.com/Getbeans/Beans/issues/36

Beans existing editor stylesheet:
![beans-default-editor-stylesheet](https://user-images.githubusercontent.com/5323259/44994427-e4049800-af9e-11e8-86f4-63474bbb0793.jpg)

Twenty Sixteen Editor Stylesheet:
![twenty-sixteen-stylesheet](https://user-images.githubusercontent.com/5323259/44994435-f252b400-af9e-11e8-8c0a-a2a064814062.jpg)

The Twenty Sixteen editor stylesheet looks a lot better then the default Beans editor stylesheet.
It would be good to change the default Beans editor stylesheet.

**Comment by christophherr** [comment]
Thank you for opening the issue, @paaljoachim.

My first thought would be to wait for Gutenberg but if anybody wants to work on this in the meantime, please go ahead.

**Comment by paaljoachim** [comment]
Yeah we might as well just wait for Gutenberg. I wrote a tutorial for this as well..:)

**Comment by ibes** [comment]
What is connected to the editor styles?

Would it be possible to just copy paste the eg Twenty Sixteen style sheet?
Or otherwise check a couple of ready existing ones and pull the best in?
